Framework/include/QualityControl/TaskInterface.h

```diff
@@ -62,6 +70,8 @@ class TaskInterface
   /// Move assignment operator
   TaskInterface& operator=(TaskInterface&& other) /* noexcept */ = default; // error with gcc if noexcept
 
+  virtual void initializeBase() final;
```

Comment:
as this is only dealing with ccdb I would rename it to "loadCcdb".

Reply:
The idea was to put everything that needs to be initialized in the parent class there, even if now it is only ccdb api. That being said, I don't mind it being `loadCcdb`